a_tutorial_of_shiny.Rmd

```diff
@@ -0,0 +1,452 @@
+# Creating Interactive Data Visualization Using Shiny
+
+Duanyue Yun, Boyu Liu
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+In this tutorial, we will use the cars dataset as an example to wall through the process of building a shiny app. The cars dataset contains various information about a particular car.
+
+```{r, eval=FALSE}
+cars_info <- read.csv("cars.csv")
+```
+
+![](resources/shiny_tutorial/carsdata.png)
+
+# Part 1 How to Build a Shiny App
```

Comment:
don't use a single hashtag header

Reply:
I made a pull request again but I don't know why it failed.